lib/emis/military_information_service_v2.rb

```diff
@@ -10,6 +10,7 @@ class MilitaryInformationServiceV2 < Service
 
     create_endpoints(
       [
+        [:get_deployment, 'deploymentRequest', 'V2'],
```

Comment:
Has adding a second endpoint to v2 brought up any concerns with the way we are currently handling the versioning of eMIS dependencies?

Reply:
Can you say more on this question? Are you concerned that we're duplicating too much, or something else?

Reply:
No, I don't have any particular concern, I'm just wondering if you ran into any pain points in adding to v2 or saw anything that might be troubling as more endpoints get added.

Reply:
It's pretty straightforward. You add an extra response and a model or two for each new method. That only feels a bit bad in this case because of the almost exact duplication between the V1 and V2 responses and models. @dbrown-va and I felt that the duplication is a fair trade-off to make to keep the different major versions of the API logically separate.